File: testsuite/matlab_and_octave/util/fix_classifier_name_inconsistency.m
function y = fix_classifier_name_inconsistency (cname)
	cname=upper(cname);

	if findstr('LIBSVM', cname)
		if length(cname) > length('LIBSVM')
			pos=findstr('LIBSVM', cname);
			y=strcat('LIBSVM_', cname(pos+6:end));
			return;
		end
	end

	y=cname;

File: testsuite/r_static/util/fix_classifier_name_inconsistency.R
fix_classifier_name_inconsistency <- function(cname) {
	cname <- toupper(cname)

	if (regexpr('LIBSVM', cname)>0) {
		if (nchar(cname)>nchar('LIBSVM')) {
			ending <- strsplit(cname, 'LIBSVM')[[1]][2]
			return(paste('LIBSVM_', ending, sep=''))
		}
	}

	return(cname)
}
